app/src/main/java/tech/relaycorp/cogrpc/server/TLSCertificateGenerator.kt

```diff
@@ -0,0 +1,100 @@
+package tech.relaycorp.cogrpc.server
+
+import org.bouncycastle.asn1.x500.X500Name
+import org.bouncycastle.asn1.x500.X500NameBuilder
+import org.bouncycastle.asn1.x500.style.BCStyle
+import org.bouncycastle.asn1.x509.Extension
+import org.bouncycastle.asn1.x509.GeneralName
+import org.bouncycastle.asn1.x509.GeneralNamesBuilder
+import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo
+import org.bouncycastle.cert.X509CertificateHolder
+import org.bouncycastle.cert.X509v3CertificateBuilder
+import org.bouncycastle.crypto.params.AsymmetricKeyParameter
+import org.bouncycastle.crypto.util.PrivateKeyFactory
+import org.bouncycastle.operator.ContentSigner
+import org.bouncycastle.operator.DefaultDigestAlgorithmIdentifierFinder
+import org.bouncycastle.operator.DefaultSignatureAlgorithmIdentifierFinder
+import org.bouncycastle.operator.bc.BcRSAContentSignerBuilder
+import org.bouncycastle.util.encoders.Base64
```

Comment:
I discovered this little gem by chance... Since we have to use BC anyway, what do you think if I replace the use of the Apache library with this?

Reply:
Fine by me